pro_tes/ga4gh/tes/task_runs.py

```diff
@@ -67,12 +67,13 @@ def create_task(  # pylint: disable=too-many-statements,too-many-branches
 
         Args:
             **kwargs: Additional keyword arguments passed along with
-                             request and start_time.
+                             request.
         Returns:
             Task identifier.
         """
         payload: Dict = deepcopy(request.json)
         db_document: DbDocument = DbDocument()
+        start_time = datetime.now().strftime("%m-%d-%Y %H:%M:%S")
```

Comment:
Put this even higher. It should be the first call in the method.

Reply:
This is fixed already in the latter commits, (my mistake I just addressed this change with the wrong commit message that was 'fixed linting').